build-dev-wheel

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Should we add the SPDX identifier to the top of this file too?

Reply:
Yup, it should. I will add it.